db/migrations/0003_alter_order_created_at.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-12-15 21:43
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_user_order_ticket_alter_movie_actors_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='created_at',
+            field=models.DateTimeField(default=django.utils.timezone.now),
```

Comment:
The `created_at` field should use `auto_now_add=True` to automatically set the field during creation, as specified in the task requirements. The current implementation using `default=django.utils.timezone.now` does not align with this requirement.